test.py

```diff
@@ -0,0 +1,147 @@
+print("\n1:")
+# 1. Create a function that reverses a string but does not use any built in Python string functions. The function will take a string and return the reversed string
+
+# example:
+# string = "Good morning"
+print("string = Good morning")
+# answer = "gninrom dooG"
+
+
+string = "Good morning"
+answer = string[::-1]
+
+print("reverse string =", answer)
+
+
+# OR!
+
+
+def reverseString(string):
+    reverseString = ""
+    for eachLetter in string:
+        reverseString = eachLetter + reverseString
+    return reverseString
+
+
+print("reverse string =", reverseString(string))
+
+
+print("\n2:")
+# 2. Given an unsorted array of numbers, find the largest and smallest numbers in the array.
+# Special Rule: You cannot sort the array to accomplish this task.
+
+# Example:
+# array = [ 5, 8, 1, 19, 7, 4, 6 ]
+print("array = [ 5, 8, 1, 19, 7, 4, 6 ]")
+# answer = Largest: 19 Smallest: 1
+
+
+array = [5, 8, 1, 19, 7, 4, 6]
+
+largeNum = array[0]
+for num in array[1:]:
+    if num > largeNum:
+        largeNum = num
+# print(largeNum)
+
+smallNum = array[0]
+for num in array[1:]:
+    if num < smallNum:
+        smallNum = num
+# print(smallNum)
+
+print("from array; Largest #:", largeNum, "Smallest #:", smallNum)
+
+
+# OR!
+
+
+def findLargeNum(numbers):
+    largeNum = numbers[0]
+    for number in numbers[1:]:
+        if number > largeNum:
+            largeNum = number
+    # print(largeNum)
+    return largeNum
+
+
+def findSmallNum(numbers):
+    smallNum = numbers[0]
+    for number in numbers[1:]:
+        if number < smallNum:
+            smallNum = number
+    # print(smallNum)
+    return smallNum
+
+
+print("from array; Largest #:", findLargeNum(
```

Comment:
the question was originaly asking to make a single function that finds both of these and returns them. So you will want to probably make a parent function that runs both of these fuinctions and returns a string shaped like
```Largest: 19 Smallest: 1```